service.py

```diff
@@ -104,6 +105,10 @@ def run(self):
         oe.stop_service()
         service_thread.stop()
         dbus_utils.LOOP_THREAD.stop()
+        # Fix crash on shutdown
+        # See https://github.com/LibreELEC/LibreELEC.tv/issues/5645
```

Comment:
no need to place comments in code .. please remove

Reply:
Ok, it's verbose here; but in my opinion, a developer should be informed of the reason for an explicit call to the garbage collector here. It's a quite rare thing in Python to attract a questioning, especially since the problem is maybe on the side of Dbussy and that it could be fixed in the future.

How about just something like "Workaround to fix crash on shutdown"?

Reply:
`# call garbage collection to workaround a crash on shutdown` works for me, the commit message is more where I'd search for things. NB: I'd be be interested to know about any theories for dbussy issues as those should be flagged and investigated (not by me) so that workarounds can be removed.

Reply:
It's done.
Yes I agree with the need for an investigation at this level;
I have contacted the developer of dbussy, maybe he will be able to give a hint...

The issue is here:
ldo/dbussy#50

Reply:
Thanks, the dbussy developer is aware of our add-on as we've corresponded on a few things now so hopefully he has some ideas. Fixups can be squashed please and I'd ask we avoid URLs in commit messages to keeep history readable, so perhaps `call garbage collection to workaround crash on shutdown` again. I'll poke others for content review as I'm a Python n00b :)